crates/modelardb_types/src/macros.rs

```diff
@@ -107,15 +107,15 @@ macro_rules! array {
 /// ```
 /// # use std::sync::Arc;
 /// #
-/// # use arrow::array::{BinaryArray, Float32Array, UInt8Array, UInt16Array, UInt64Array};
+/// # use arrow::array::{BinaryArray, Float32Array, Int8Array, Int16Array, Int64Array};
```

Comment:
I assume `Int64Array` can be removed now that we no longer have univariate ids.

Reply:
Fixed.